packages/react/src/__tests__/jest-matcher.test.tsx

```diff
@@ -124,6 +124,14 @@ describe('toHaveCompliedCss', () => {
     expect(el).toHaveCompiledCss('color', 'blue', { target: ':active' });
   });
 
+  it('should match camelCase css vars', () => {
```

Comment:
Did you want a test for `{ '--my-snake-var': 'red' }` as well…?

Reply:
https://github.com/atlassian-labs/compiled/pull/1676